src/Infrastructure/Connector/Action/Currency/PostCurrencyCreate.php

```diff
@@ -29,16 +32,26 @@ public function getRequest(): Request
             HttpRequest::METHOD_POST,
             $this->getUri(),
             $this->buildHeaders(),
-            $this->buildBody()
+            $this->buildBody(),
         );
     }
 
     /**
-     * @return null
+     * @param string|null $content
+     *
+     * @return string|null
+     *
+     * @throws \JsonException
      */
     public function parseContent(?string $content)
     {
-        return null;
+        if (!$content) {
+            return null;
+        }
+
+        $data = json_decode($content, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
'ext-json' is missing in composer.json - you can add it there

Reply:
Please consider taking advantage of JSON_THROW_ON_ERROR flag for this call options.